drivers/bluetooth/hci/Kconfig

```diff
@@ -30,6 +30,19 @@ config BT_H5
 	  Bluetooth three-wire (H:5) UART driver. Implementation of HCI
 	  Three-Wire UART Transport Layer.
 
+config BT_RPMSG
+	bool "RPMsg HCI"
+	select IPM
+	select IPM_NRFX
+	select NRFX_IPC
```

Comment:
@kapi-no this selection is redundant here, isn't it? I think IPM_NRFX selects NRFX_IPC

Reply:
As is, BT_RPMSG seems Nordic-specific, but this is not reflected in the nomenclature; has this been attended somehow?

Reply:
removed redundant NRFX_IPC

Reply:
The select part of the config is placed now under BT_RPMSG_NRF53 Kconfig option